src/main/identity/id-attribute-type-service.js

```diff
@@ -1,20 +1,57 @@
 'use strict';
 import fetch from 'node-fetch';
+import { Logger } from 'common/logger';
+import RefParser from 'json-schema-ref-parser';
 import IdAttributeType from './id-attribute-type';
+import { IdAttributeSchema } from './id-attribute-schema';
 
+const log = new Logger('id-attribute-type-service');
 const airtableBaseUrl = 'https://alpha.selfkey.org/marketplace/i/api/';
-
+export const JSON_SCHEMA_EXPIRTION_TIME = 1000 * 60 * 60 * 5; // 5h
+export const JSON_SCHEMA_URL = 'http://platform.selfkey.org/schema/attribute';
```

Comment:
@sk91 I would also move `http://platform.selfkey.org/schema/` to a global config and just append `attribute` to it. Thoughts?

Reply:
Something to remember: not all schemas will start with `http://platform.selfkey.org/schema/`

One of the reasons we're going down this path is so that third parties can define ID attributes for the wallet.

I don't even think this should be a configuration value. We should seed a schema list somewhere (whether a cache table or some other data store) with a list of our standard schemas, but they should be stored as fully-qualified URLs - otherwise we need to add special handling for our own schemas, which isn't a great design decision.

Reply:
@rodrigopavezi @dfcowell that's exactly why schema has "jsonSchemaUrl" property. JSON_SCHEMA_URL is used only if it is not provided. 

So @rodrigopavezi no, I don't think we should have it as configuration, it will be refactored away in one of the next pull requests (when we start receiving schema url together with id-attribute-type)

@dfcowell @ben has committed a json list of schemas to platforms if I remember correctly. In next iterations, we will start using it instead of the air-table

Reply:
@sk91 @dfcowell I see now. makes sense. cheers